### DISCUSSION TITLE
请问这个项目要怎么使用呢？

### DISCUSSION THREAD

**Discussion by daifuyang** [alibaba/designable#35] [Q&A]
如题，项目拉下来该如何运行呢？现在打开显示的是一个dumi首页。

> root@ start:playground D:\workspace\designable
> cd packages/playground && npm run start

Debugger attached.

> @designable/playground@0.4.25 start D:\workspace\designable\packages\playground
> webpack-dev-server --config config/webpack.dev.ts

Debugger attached.
Error: Cannot find module '@formily/reactive'
Require stack:
- D:\workspace\designable\packages\playground\config\webpack.base.ts
- D:\workspace\designable\packages\playground\config\webpack.dev.ts
- D:\workspace\designable\node_modules\webpack-cli\bin\utils\convert-argv.js
- D:\workspace\designable\node_modules\webpack-dev-server\bin\webpack-dev-server.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.resolve (internal/modules/cjs/helpers.js:94:19)
    at D:\workspace\designable\packages\playground\config\webpack.base.ts:14:27
    at Array.reduce (<anonymous>)
    at getAlias (D:\workspace\designable\packages\playground\config\webpack.base.ts:10:49)
    at Object.<anonymous> (D:\workspace\designable\packages\playground\config\webpack.base.ts:51:12)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Module.m._compile (D:\workspace\designable\node_modules\ts-node\src\index.ts:1056:23)
    at Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Object.require.extensions.<computed> [as .ts] (D:\workspace\designable\node_modules\ts-node\src\index.ts:1059:12)
Waiting for the debugger to disconnect...
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @designable/playground@0.4.25 start: `webpack-dev-server --config config/webpack.dev.ts`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @designable/playground@0.4.25 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     D:\SoftWare\node\node_cachel\_logs\2021-08-06T05_43_51_998Z-debug.log
Waiting for the debugger to disconnect...
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! root@ start:playground: `cd packages/playground && npm run start`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the root@ start:playground script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     D:\SoftWare\node\node_cachel\_logs\2021-08-06T05_43_52_092Z-debug.log
Waiting for the debugger to disconnect...

**Comment by MaskHao** [comment]
提示你了啊，进入文件夹 packages/playground  ，然后运行 npm run start

**Comment by daifuyang** [comment]
谢谢，已经跑起来了。项目非常棒！
